### PR TITLE
Frontend - Adding markup for reviews for Google rich cards.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**
## What does this change?
The associated changes add structured data markup for articles containing structured reviews for games, films and restaurants.

## What is the value of this and can you measure success?
By adding structured data to our archive we aim to improve our google ranking for archive content and increase the number of page views. Current metrics are:

- ranked 7.5 on average
- 50 millions page views per year

## Does this affect other platforms - Amp, Apps, etc?
This may effect AMP.

## Screenshots
None.

## Tested in CODE?
AMP validation tested locally for reviews of each type.

Please see https://github.com/guardian/frontend/pull/15556 for further information.
@stephanfowler @mchv

### CODE REVIEW

File: common/app/views/fragments/atoms/structureddata/gameReview.scala.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "worstRating":@review.data.rating.minRating.toInt,
+    "bestRating":@review.data.rating.maxRating.toInt,
+    "ratingValue": @review.data.rating.actualRating.toInt
+  },
+
+  "name": "@entity.title",
+  "author": {
+    "@@type": "Person",
+    "name": "@review.data.reviewer"
+  },
```
Comment (mchv) [inline common/app/views/fragments/atoms/structureddata/gameReview.scala.html]:
```scala
 "author": @{fragments.atoms.structureddata.person(review.data.reviewer)} ,
```

**Comment by mchv** [comment]
This looks good to me 👍 

I have added a small improvement suggestion but I think it is better to merge as it is and create a new PR for improvement

**Comment by stephanfowler** [comment]
As just discussed with Mariot, worth checking that all this validates in AMP.

**Comment by LATaylor-guardian** [comment]
@stephanfowler Thanks. How do I do this?

**Comment by stephanfowler** [comment]
Try this, and look in the console:
Local: `localhost:9000/an-article?amp#development=1`
Prod: `amp.theguardian.com/an-article#development=1`

**Comment by LATaylor-guardian** [comment]
@stephanfowler Thanks, I've checked a couple of each type (game, restaurant and film) and they validated successfully.

**Pull request merged**